ui/pages/import-token/import-token.component.js

```diff
@@ -133,6 +135,12 @@ class ImportToken extends Component {
     );
   }
 
+  hasWarning() {
```

Comment:
It looks like this method is not used... can it be deleted?

Reply:
deleted